### PR TITLE
Add support to create, update and delete uplink sets

### PR DESCRIPTION
**Pull request by balestrinc**

### CODE REVIEW

File: hpOneView/resources/networking/uplink_sets.py [hunk 1]
```diff
@@ -101,3 +105,56 @@ def get_by(self, field, value):
...
+
+        Args:
+            resource: Resource to delete or the resource ID
+            force:
+                 If set to true the operation completes despite any problems with
+                 network connectivity or errors on the resource itself. The default is false.
+            timeout:
+                Timeout in seconds. Wait task completion by default. The timeout does not abort the operation in
+                OneView, just stops waiting for its completion.
+
```
Comment (marikrg) [inline hpOneView/resources/networking/uplink_sets.py]: The function is returning a value, but the `Returns` section is missing from the docstring.

File: examples/uplink_sets.py [hunk 2]
```diff
@@ -39,33 +38,50 @@
...
-try:
-    uplink_set = oneview_client.uplink_sets.get(uplink_set_uri)
-    pprint(uplink_set)
-except HPOneViewException as e:
-    print(e.msg)
+uplink_set = oneview_client.uplink_sets.get(created_uplink_set['uri'])
+pprint(uplink_set)
+
+# Delete the recently created uplink set
+retorno = oneview_client.fc_networks.delete(updated_uplink_set)
```
Comment (marikrg) [inline examples/uplink_sets.py]: I suggest change that variable name.

File: hpOneView/resources/networking/uplink_sets.py [hunk 3]
```diff
@@ -101,3 +105,56 @@ def get_by(self, field, value):
...
         """
         return self._client.get_by(field, value)
+
+    def create(self, resource, timeout=-1):
+        """
+        Creates an uplink set.
+
+        Args:
+            resource: dict object to create
+            imeout:
```
Comment (marikrg) [inline hpOneView/resources/networking/uplink_sets.py]: Missing letter "t" in the word timeout.

**Comment by balestrinc** [comment]
@marikrg  done

**Pull request merged**